conf/json/cc009c-schema.json

```diff
@@ -14589,9 +14589,6 @@
     },
     "n1:MessageTypes": {
       "description": "The list of codes used to identify the type of messages exchanged on the Common Domain and on the External Domain (e.g. CD001B, CD302A, CC515C, CD801A).The values of this codelist are also available and updated in CS/RD2.",
-      "enum": [
-        "CC009C"
-      ],
```

Comment:
If we're going to do this, we should at least add a pattern in its place to make it much closer to 

```
"pattern": "^C[CD]\\d{3}C$"
```

or make the enum hold all valid values as per the XSD.

Reply:
to be consistent with xsd - added all valid values to the list